packages/package/test/index.js

```diff
@@ -127,6 +127,10 @@ test('create standard package with typescript', async () => {
 	await test_make_package('typescript');
 });
 
+test('create standard package with typescript using nodenext', async () => {
```

Comment:
What do you mean by "default nodenext"? I don't know what that means 😄 It's the default now, but it could change later on. I think it's fine like this.

Reply:
I think the fixture names are a bit confusing at the moment because the one with the basic name (`typescript`) overrides the default settings and the one with extra description (`typescript-nodenext`) uses the default settings, which isn't really what I'd expect from just looking at the names. It's less a comment about this one and more a comment about the existing one, but GitHub won't let me comment on a non-changed line :laughing: Maybe we could change the other one to `typescript-esnext` for consistency?